example/lib/main.dart

```diff
@@ -42,88 +52,93 @@ class _MyHomePageState extends State<MyHomePage> {
   UhstSocket? client;
   final TextEditingController _hostIdController = TextEditingController();
 
-  @override
-  void initState() {
-    super.initState();
-  }
-
-  Future<void> initHost() async {
+  Future<void> connectHost() async {
```

Comment:
why rename?

Reply:
to unify functions and make them more understandable:
connectHost
disconnectHost

Reply:
I find it more confusing, maybe startHosting/stopHosting ?

Reply:
agreed, it sounds nicer, will fix it